src/timesync/timesyncd-manager.c

```diff
@@ -1221,3 +1223,58 @@ static int manager_save_time_and_rearm(Manager *m, usec_t t) {
 
         return 0;
 }
+
+static const char* ntp_server_property_name[] = {
+        "SystemNTPServers",
+        "FallbackNTPServers",
+        "LinkNTPServers",
+        "RuntimeNTPServers",
+};
+
+static int on_deferred_ntp_server(sd_event_source *s, void *userdata) {
+        Manager *m = ASSERT_PTR(userdata);
+        int r;
+        log_warning("doujb: here come to emit the ntp server signal function");
+        m->deferred_ntp_server_event_source = sd_event_source_disable_unref(m->deferred_ntp_server_event_source);
+        for(int type = SERVER_SYSTEM; type <= SERVER_RUNTIME; type++)
+        {
```

Comment:
please follow coding style, i.e. we place opening bracket on same line as `for` is placed